docs/en/sql-reference/statements/create/view.md

```diff
@@ -36,6 +36,18 @@ This query is fully equivalent to using the subquery:
 SELECT a, b, c FROM (SELECT ...)
 ```
 
+## Parameterized View
+This is similar to normal view but can be created with parameter instead of literals and can be used as table functions by substituting the values of the parametes.
```

Comment:
```suggestion
## Parameterized View

Parametrized views are similar to normal views but can be created with parameters which are not resolved immediately. These views can be used with table functions which specify the name of the view as function name and the parameter values as its arguments.
```
Not necessary to change this comment, it is just a suggestion how it can be improved a little bit.

Reply:
addressed it